index.html

```diff
@@ -29,29 +35,31 @@
         <!-- you can nest rows inside of columns! -->
         <!-- our board has 3 rows, each with 3 columns inside -->
         <div class="row">
-          <div class="col-xs-4 box"></div>
-          <div class="col-xs-4 box"></div>
-          <div class="col-xs-4 box"></div>
+          <div class="col-xs-4 box"id="a"></div>
+          <div class="col-xs-4 box"id="b"></div>
+          <div class="col-xs-4 box"id="c"></div>
         </div>
 
         <div class="row">
-          <div class="col-xs-4 box"></div>
-          <div class="col-xs-4 box"></div>
-          <div class="col-xs-4 box"></div>
+          <div class="col-xs-4 box"id="d"></div>
+          <div class="col-xs-4 box"id="e"></div>
+          <div class="col-xs-4 box"id="f"></div>
         </div>
 
         <div class="row">
-          <div class="col-xs-4 box"></div>
-          <div class="col-xs-4 box"></div>
-          <div class="col-xs-4 box"></div>
+          <div class="col-xs-4 box"id="g"></div>
+          <div class="col-xs-4 box"id="h"></div>
+          <div class="col-xs-4 box"id="i"></div>
         </div>
       </div>
     </div>
   </div>
   
   <!-- jquery -->
+<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.0.0-alpha1/jquery.min.js"></script>
```

Comment:
Did you intend to use jQuery 3 alpha? We're using 1.10 or 1.11 elsewhere and noticed a few people using 3.

It isn't a bad thing but may cause issues if you weren't intending to use the alpha version.